types/fh-sync.d.ts

```diff
@@ -0,0 +1,314 @@
+// Type definitions for fh-sync
+// Project: https://github.com/feedhenry/fh-sync
+// Maintainer feedhenry-dev@redhat.com
+
+declare module SyncCloud {
+  export namespace Sync {
+    /**
+     * Options used to initialize Sync Server
+     */
+    interface SyncGlobalOptions {
+      /** @type {Number} how often pending workers should check for the next job, in ms. Default: 1 */
```

Comment:
All this @type stuff is redundant, since the field already has a type associated with it. In fact, VS Code doesn't seem to parse the doc strings if they start with @type.